nevergrad/optimization/test_optimizerlib.py

```diff
@@ -640,8 +640,8 @@ def test_bo_ordering() -> None:
         ("NGOpt8", 3, 1, False, 100, ["OnePlusOne", "OnePlusOne"]),
         ("NGOpt8", 3, 1, False, 200, ["SQP", "SQP"]),
         ("NGOpt8", 3, 1, True, 1000, ["SQP", "monovariate", "monovariate"]),
-        (None, 3, 1, False, 1000, ["CMA", "CMA"]),
-        (None, 3, 20, False, 1000, ["MetaModel", "MetaModel"]),
+        (None, 3, 1, False, 1000, ["CMA", "OnePlusOne"]),
+        (None, 3, 20, False, 1000, ["MetaModel", "OnePlusOne"]),
```

Comment:
@teytaud there's an interesting change here:
before this PR, the splitoptimizer was not able to figure out that the Array representing the softmax weights introduced noise. Now it does, so it changes from CMA/Metamodel to OnePlusOne. Does it make sense?